### DISCUSSION TITLE
r68s配置静态路由后，路由器错误

### DISCUSSION THREAD

**Discussion by Constantinefun** [coolsnowwolf/lede#9604] [R68S / 电犀牛专区]
配置eth3的静态路由，路由表会错误识别为eth2口
![image](https://user-images.githubusercontent.com/47021286/174416877-602f3435-d9e8-4d5c-9685-38e7cb3ce1bb.png)
![image](https://user-images.githubusercontent.com/47021286/174416886-7728236e-7c8e-440d-8e56-f691d8008723.png)
![image](https://user-images.githubusercontent.com/47021286/174416899-d69e4d90-7abe-4eca-bce6-fba3acbe9f94.png)

**Comment by tianshicad** [comment]
路由界面可以进去吗？
